universal-login-example/src/components/ContentContainer.js

```diff
@@ -33,6 +34,15 @@ class ContentContainer extends Component {
     window.scrollTo(0, 0);
   }
 
+  async loadIdentity() {
+    const {storageService, identityService} = this.props.services;
+    let identity = await storageService.checkForIdentity();
+    if (identity) {
+      identityService.identity = identity;
+      this.setView('MainScreen');
+    }
+  }
```

Comment:
There should be no logic in Reacgt components whatsoever. All the logic should be in services.
In particular, IdentityService might have a StorageService as a dependency (passed in constructor during build time).
All persistence should be implemented in StorageService and covered with roundtrip tests. 
In the top of that, we have IdentityService which implements logic like getting identity from storage or from sdk and updating state based on those.